tpp-app/ios/tppapp/Images.xcassets/icons/onboard_bar.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="60" height="6" viewBox="0 0 60 6" fill="none" xmlns="http://www.w3.org/2000/svg">
+<circle cx="3" cy="3" r="3" transform="rotate(-180 3 3)" fill="#D9D9D9"/>
+<circle cx="19" cy="3" r="3" transform="rotate(-180 19 3)" fill="#D9D9D9"/>
+<rect x="60" y="6" width="28" height="6" rx="3" transform="rotate(-180 60 6)" fill="#73C7B7"/>
+</svg>
```

Comment:
Can you upload and use the other onboarding progress bars as well? 
this is the one that currently appears on all onboarding screens: 
<img width="178" alt="Screen Shot 2022-03-28 at 4 38 10 PM" src="https://user-images.githubusercontent.com/35851484/160483016-e4749337-cda9-455f-b927-110f8b2101c8.png">
But the first page should use this one: 
<img width="260" alt="Screen Shot 2022-03-28 at 4 39 22 PM" src="https://user-images.githubusercontent.com/35851484/160483183-e554a15b-f091-4707-a52f-d2876db86aa6.png">
And second page use this one: 
<img width="257" alt="Screen Shot 2022-03-28 at 4 39 52 PM" src="https://user-images.githubusercontent.com/35851484/160483245-ff61a873-c57f-4caa-8d94-f817508111da.png">